python/pylibvw.cc

```diff
@@ -480,12 +556,18 @@ float ex_get_simplelabel_label(example_ptr ec) { return ec->l.simple.label; }
 float ex_get_simplelabel_weight(example_ptr ec) { return ec->l.simple.weight; }
 float ex_get_simplelabel_initial(example_ptr ec) { return ec->l.simple.initial; }
 float ex_get_simplelabel_prediction(example_ptr ec) { return ec->pred.scalar; }
+void  ex_set_simplelabel_prediction(example_ptr ec, float fl) { ec->pred.scalar = fl; }
 float ex_get_prob(example_ptr ec) { return ec->pred.prob; }
 
 uint32_t ex_get_multiclass_label(example_ptr ec) { return ec->l.multi.label; }
 float ex_get_multiclass_weight(example_ptr ec) { return ec->l.multi.weight; }
 uint32_t ex_get_multiclass_prediction(example_ptr ec) { return ec->pred.multiclass; }
 
+float ex_get_scalar(example_ptr ec)
```

Comment:
Isn't this the same as ex_get_simplelabel_prediction ?

Reply:
emphasis on SET

Reply:
good point, initially I misread as get/set issue.